.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+<!--
+Thanks for contributing to Jupyter Scheduler!
+Please fill out the following items to submit a pull request.
+Please refer to our contributor's guide for more information on installation and usage:
+https://jupyter-scheduler.readthedocs.io/en/latest/contributors/index.html
+-->
+
+## References
+
+<!-- Note issue numbers this pull request addresses (should be at least one, see contributing guidelines above). -->
```

Comment:
Change "Note issue numbers  ... " to "Note the issue numbers ...". Also insert a "the" before contributing guidelines.